lib/elixometer.ex

```diff
@@ -262,7 +262,7 @@ defmodule Elixometer do
   If the value of the `:reset_seconds` option is greater than zero, the counter will be reset
   automatically at the specified interval.
   """
-  @spec update_counter(bitstring, float, [reset_seconds: nil | pos_integer]) :: :ok
+  @spec update_counter(String.t, number, [reset_seconds: nil | integer]) :: :ok
```

Comment:
`number` -> `integer` as counter only supports integers in `:ets.update_counter`

Reply:
Thanks